### ISSUE TITLE
Widget is a gray box

### ISSUE THREAD

**Issue by smichel17** [LibreShift/red-moon#195]
> the widget icon has disappeared too and instead it's just a grey block. Not as crippling but a bit annoying.

**Comment by smichel17** [comment]
@consideredwords Have you tried removing and re-creating the widget?

**Comment by consideredwords** [comment]
Yes I have thanks, and I when I downgraded RedMoon to the previous version I had installed where it worked (3.0) it was still grey.

**Comment by consideredwords** [comment]
Here is what the widget looks like on my homescreen and in the widget selection tool

![screenshot_20170930-111018](https://user-images.githubusercontent.com/20317088/31045135-a9ae9e0a-a5d4-11e7-88fd-70b7e70e6c75.png)
![screenshot_20170930-111024](https://user-images.githubusercontent.com/20317088/31045136-a9ba7cf2-a5d4-11e7-86a3-907fcf8982a8.png)

**Comment by smichel17** [comment]
That is the intended widget, and it's been around for a while. However, there should also be a home screen *shortcut*, which seems to be missing for you.
